Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -255,10 +282,11 @@ private class AccountStatusSetting: WithAccountSetting {
                 viewController.url = cs?.URL
             case .None, .NeedsUpgrade:
                 // In future, we'll want to link to /settings and an upgrade page, respectively.
-                return
+                let cs = NSURLComponents(URL: account.configuration.forceAuthURL, resolvingAgainstBaseURL: false)
```

Comment:
Same bug, this just allows the user to tap the account row (e.g. brymunar@gmail.com) to go back to the login page just in case something funky comes up and they need to do something (they changed their password, etc. not really familiar on the specifics, but this is to provide a way out of some sync issues that might occur--look to tecgirl for more answers, I think she's the once that wanted this action to be implemented w/ Sync button)